src/MeshlessFV/MeshlessFVTree.cpp

```diff
@@ -782,18 +764,17 @@ void MeshlessFVTree<ndim,ParticleType,TreeCell>::UpdateGodunovFluxes
       for (int jj=0; jj<Nneib; jj++) {
         i = neibpart[jj].iorig;
         if (!neibpart[jj].flags.is_mirror()) {
-	        if (neibpart[jj].flags.check_flag(active))
```

Comment:
I don't think this makes a difference.  The active checks are made else where.  This is simply adding all the contributions together.